CHANGELOG.md

```diff
@@ -1,15 +1,9 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
+## 1.3.1 (2022-04-05)
 
 ### Bugs Fixed
 
```

Comment:
Should we mentioned the header value changed?

Reply:
Do you mean the text that you added from https://github.com/Azure/azure-sdk-for-c/pull/2154, or something else?

Merging main into this branch brought it, so we should be good.

Reply:
Marking this as resolved and merging the PR. If you were thinking of something else, lmk, and I will update it.